.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -r requirements-dev.txt
+    - name: Run tests
+      run: |
+        make cov
+        python setup.py check -rms
```

Comment:
Not sure if this is still relevant. I suggest exploring (in a separate PR, of course) if all the checks embedded into this command can be replaced by `twine check`.